alpha/declcfg/load.go

```diff
@@ -118,7 +118,7 @@ func extractCSV(objs []string) string {
 	return ""
 }
 
-func readYAMLOrJSON(r io.Reader) (*DeclarativeConfig, error) {
+func LoadReader(r io.Reader) (*DeclarativeConfig, error) {
```

Comment:
This should have some GoDoc comments that describe what it does. And part of that description should mention the fact that path references will not be de-referenced and that it's up to callers to do that de-referencing if necessary.